### PR TITLE
Add parameters back to ConsLaborModel and fix up to match new distribution code

### PR DESCRIPTION
**Pull request by MridulS**
addresses https://github.com/econ-ark/HARK/issues/721, https://github.com/econ-ark/HARK/issues/684, https://github.com/econ-ark/HARK/issues/681

### CODE REVIEW

**Comment by MridulS** [comment]
As this runs black on the codebase it may have added to much to the files changed tab, so have a look at the specific commits

https://github.com/econ-ark/HARK/pull/729/commits/e3d6f2dd0f0f629c225531022ed40f3842bef265 (changes to ConsLaborModel)

and 

https://github.com/econ-ark/HARK/pull/729/commits/5c43e0287b6151417cf563ad448a530b645f5bea (adds tests)

to review this PR. :)

File: HARK/ConsumptionSaving/tests/test_ConsLaborModel.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    LaborIntMargConsumerType,
+    init_labor_lifecycle,
+)
+import unittest
+
+
+class test_LaborIntMargConsumerType(unittest.TestCase):
+    def setUp(self):
+        self.model = LaborIntMargConsumerType()
+        self.model_finte_lifecycle = LaborIntMargConsumerType(**init_labor_lifecycle)
```
Comment (sbenthall) [inline HARK/ConsumptionSaving/tests/test_ConsLaborModel.py:11]: finte -> finite ?

**Comment by sbenthall** [comment]
I'm loving the blackify style.

**Comment by sbenthall** [comment]
This looks good to me except for the typo in the test.

**Pull request merged**